etcd/etcd-backup.adoc

```diff
@@ -1,7 +1,149 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="etcd-backup"]
 include::_attributes/common-attributes.adoc[]
-= Backing up etcd
+= Backing up and restoring etcd data
 :context: etcd-backup
 
-// This assembly will contain modules to provide information about backing up and restoring etcd.
\ No newline at end of file
+As the key-value store for {product-title}, etcd persists the state of all resource objects.
```

Comment:
I'm also realizing now that there's no TOC on the right hand side of the page:
```suggestion
toc::[]

As the key-value store for {product-title}, etcd persists the state of all resource objects.
```

Reply:
Good catch!